src/StorageSync/StorageSync/StorageSync.csproj

```diff
@@ -12,7 +12,7 @@
 
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="2.8.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="3.0.0" />
```

Comment:
You will need to release this SDK before we can merge this PR.

Reply:
Agreed. I'll make the change to consume the new SDK official when it is released.

Reply:
The SDK has been officially published now. I removed the pkg from LocalFeeds.
https://www.nuget.org/packages/Microsoft.Azure.Management.StorageSync/3.0.0